content/en/apps/guides/hosting/monitoring/setup.md

```diff
@@ -248,4 +248,5 @@ All CHT metrics in Prometheus:
 | `cht_outbound_push_backlog_count`     | Gauge   |                                                   | Number of changes yet to be processed by Outbound Push.                                                                                                                                                         |
 | `cht_replication_limit_count`         | Gauge   |                                                   | Number of users that exceeded the replication limit of documents.                                                                                                                                               |
 | `cht_sentinel_backlog_count`          | Gauge   |                                                   | Number of changes yet to be processed by Sentinel.                                                                                                                                                              |
+| `cht_api_*`                           | Any     |                                                   | Node.js metrics for CHT API as detailed by [prometheus-api-metrics](https://www.npmjs.com/package/prometheus-api-metrics) including default metrics and GC metrics. As of 4.3.0. |
```

Comment:
I think this is too abstract and suspect it's more helpful to an administrator to focus on the API server metrics like so:

```suggestion
| `cht_api_*`                           | Any     |                                                   | API server metrics (see [prometheus-api-metrics](https://www.npmjs.com/package/prometheus-api-metrics)). Requires CHT Core 4.3.0 or later. Includes stats like server response time in seconds and response size in bytes. |
```